astropy/wcs/tests/test_wcsprm.py

```diff
@@ -430,21 +431,28 @@ def test_fix2():
         'cdfix': 'No change',
         'cylfix': 'No change',
         'obsfix': 'No change',
-        'datfix': ("Set MJD-OBS to 51543.000000 from DATE-OBS.\n"
-                   "Changed DATE-OBS from '31/12/99' to '1999-12-31'"),
+        'datfix': (
+            "Set MJD-OBS to 51543.000000 from DATE-OBS.\n"
+            "Changed DATE-OBS from '31/12/99' to '1999-12-31'"
+        ),
         'spcfix': 'No change',
         'unitfix': 'No change',
-        'celfix': 'No change'}
+        'celfix': 'No change',
+    }
     version = wcs._wcs.__version__
     if Version(version) <= Version("5"):
         del fix_ref['obsfix']
         fix_ref['datfix'] = "Changed '31/12/99' to '1999-12-31'"
 
     if Version(version) >= Version('7.3'):
-        fix_ref['datfix'] = "Set DATEREF to '1858-11-17' from MJDREF.\n" + fix_ref['datfix']
+        fix_ref[
+            'datfix'
+        ] = f"Set DATEREF to '1858-11-17' from MJDREF.\n{fix_ref['datfix']}"
 
     elif Version(version) >= Version('7.1'):
-        fix_ref['datfix'] = "Set DATE-REF to '1858-11-17' from MJD-REF.\n" + fix_ref['datfix']
+        fix_ref[
+            'datfix'
+        ] = f"Set DATE-REF to '1858-11-17' from MJD-REF.\n{fix_ref['datfix']}"
```

Comment:
I think this is less clearer or readable than the one on line 481 but may be it's just me.

Reply:
I like the `f`-string but
```python
fix_ref[
    'datfix'
]
```
is bad, IMO.

Reply:
In this case I feel like the split of the key reference made it less readable than it was before.